src/content.js

```diff
@@ -310,18 +310,19 @@ export function removeFormattingElem (host, range, elem) {
   })
 }
 
-export function removeFormatting (host, range, tagName) {
+export function removeFormatting (host, range, tagName, className) {
   return restoreRange(host, range, () => {
-    nuke(host, range, tagName)
+    nuke(host, range, tagName, className)
   })
 }
 
 // Unwrap all tags this range is affected by.
 // Can also affect content outside of the range.
-export function nuke (host, range, tagName) {
+export function nuke (host, range, tagName, className) {
   getTags(host, range).forEach((elem) => {
     if (elem.nodeName.toUpperCase() !== 'BR' &&
-      (!tagName || elem.nodeName.toUpperCase() === tagName.toUpperCase())) {
+      (!tagName || (elem.nodeName.toUpperCase() === tagName.toUpperCase() &&
+        (!className || elem.classList.contains(className))))) {
```

Comment:
We could use a selector here instead of a fixed class name. That way you can pass `.someclass` and also attributes `[data-someattribute]`.

Reply:
Great suggestion, thanks @marcbachmann! Commit https://github.com/livingdocsIO/editable.js/pull/228/commits/1f008690ce35f9304d63910a4577a8cc7cb14f1c.